Blockzilla/OverlayView.swift

```diff
@@ -125,6 +157,10 @@ class OverlayView: UIView {
         button.setAttributedTitle(attributedString, for: .normal)
     }
     
+    func setCurrentURL(url: String) {
```

Comment:
Change to private(set) on this var

Reply:
Can you expand on this a bit? Are you suggesting something like this (in place of making a function):

`private(set) var currentURL = ""`

If I change this, I'm unable to set currentURL from BrowserViewController. Perhaps this can just be public?

Reply:
sorry! my mistake, leave as-is